### ISSUE TITLE
[Mobile] Reflow issues for abandoned channels in Blocked Page

### ISSUE THREAD

**Issue by infinite-persistence** [lbryio/lbry-desktop#4434]
**Bug**
See image below.
Also if 4433 gets approved, the fix for this also needs to take into account the new "Remove from list" button for abandoned claims.

![image](https://user-images.githubusercontent.com/64950861/85615278-933bc300-b68e-11ea-8bd1-519b0c983ed4.png)

**Comment by infinite-persistence** [comment]
No longer relevant after #4433 `Option to remove abandoned claims from Blocked Channels page` (the correct component will now be used, which doesn't overflow).

![image](https://user-images.githubusercontent.com/64950861/86043717-a928f980-ba7b-11ea-81ed-be7acb404682.png)
